test/integration/scheduler/framework_test.go

```diff
@@ -1699,8 +1690,8 @@ func TestFilterPlugin(t *testing.T) {
 				}
 			}
 
-			if filterPlugin.numFilterCalled != 1 {
-				t.Errorf("Expected the filter plugin to be called 1 time, but got %v.", filterPlugin.numFilterCalled)
+			if filterPlugin.numFilterCalled < 1 {
```

Comment:
we can do the same, assert it is 1 in the success case, and one or more in case of failure

Reply:
true, let me update.

Reply:
Done.